chart/values.yaml

```diff
@@ -450,6 +450,10 @@ opentelemetryOperator:
           endpoint: "{{ .Release.Name }}-promscale-connector.{{ .Release.Namespace }}.svc.cluster.local:9202"
           tls:
             insecure: true
+        prometheusremotewrite:
```

Comment:
Nit: should we use `camelCase` here?

Reply:
This option is not used by us but by an upstream component. More in https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/prometheusremotewriteexporter#prometheus-remote-write-exporter